File: palindrome.py
#check if a word is a palindrome
while True:
    x = 0

    list= input("enter numbers to check if its a palindrome: ")
    first_check = (list[:])
    second_check = (list[::-1])
    if first_check == second_check:
        print( list + " is a palindrome")
    else:
        print(list + " is not a palindrome please")
    x += 1

File: javascript/palindrome.js
function isPalindrome(word){
	var len = string.length;
	for(var i = 0; i < len/2; i++){
		if (word[i] !== string[len - 1 - i]) {
			alert('not a palindrome');
		}
	}
	alert('This is a Palindorme!');
}
